components/widgets/inputs/users_emails_input.jsx

```diff
@@ -25,6 +26,48 @@ import {isGuest} from 'utils/utils';
 
 import './users_emails_input.scss';
 
+const NotifyButton = () => {
+    const NOT_STARTED = 'Not_Started';
+    const STARTED = 'Started';
+    const SUCCESS = 'Success';
+    const FAILED = 'Failed';
+
+    const [notifyStatus, setStatus] = useState(NOT_STARTED);
+    const {formatMessage} = useIntl();
+
+    const notifyFunc = async () => {
+        try {
+            setStatus(STARTED);
+            await Client4.sendOverLimitAlert();
+            setStatus(SUCCESS);
+        } catch (error) {
+            if (error) {
```

Comment:
I think this is a unnecessary if condition. You don't do any special validation, just to make sure there is an error, which I think is implicit if you are in the catch block.